src/jit.rs

```diff
@@ -20,17 +20,34 @@ use std::fmt::Error as FormatterError;
 use std::ops::{Index, IndexMut};
 
 use crate::{
-    vm::JitProgram,
     vm::{Executable, Syscall},
     call_frames::CALL_FRAME_SIZE,
-    ebpf::{self, FIRST_SCRATCH_REG, SCRATCH_REGS, STACK_REG, MM_STACK_START},
+    ebpf::{self, INSN_SIZE, FIRST_SCRATCH_REG, SCRATCH_REGS, STACK_REG, MM_STACK_START, MM_PROGRAM_START},
     error::{UserDefinedError, EbpfError},
     memory_region::{AccessType, MemoryMapping},
     user_error::UserError,
 };
 
+extern crate libc;
+
+/// eBPF Jit-compiled program.
```

Comment:
copypasta from below?

Reply:
Seems so, doesn't it belong at the beginning of the file?

Reply:
The beginning of the file could use a comment, but probably not this one

Reply:
Ah, got it. I thought you were talking about `extern crate libc;`